### ISSUE TITLE
[Screen Reader][TextField] Narrator doesn't read errorMessage in scan mode when it changes

### ISSUE THREAD

**Issue by anmohanty123** [microsoft/fluentui#23046]
I am facing this issue as well. Opening this because the other one got closed

We too face similar kind of issue with TextField. We are using `onBlur` method to show the error message when, the value entered is incorrect and the focus is out. 
But on clicking tab, focus is shifted to next focusable element and the error message which popped up is announced after announcing the element that got focused. Expected is to announce the error message immediately on tabbing out.

_Originally posted by @Bhavana-Nimma in https://github.com/microsoft/fluentui/issues/16433#issuecomment-876283476_
